ui/resizable.js

```diff
@@ -344,16 +344,16 @@ $.widget("ui.resizable", $.ui.mouse, {
 				width: this.helper.width(),
 				height: this.helper.height()
 			} : {
-				width: el.width(),
-				height: el.height()
+				width: el.outerWidth(),
+				height:el.outerHeight()
 			};
 
 		this.originalSize = this._helper ? {
 				width: el.outerWidth(),
 				height: el.outerHeight()
 			} : {
-				width: el.width(),
```

Comment:
I think the clue that there is something more going on here not addressed by the PR is that this change essentially negates the need for the `this._helper` check--the two objects are now the same! Also, you would want a test case that shows this change actually fixes a situation that was broken.